src/test/java/temp/tests/TestOpenPageFirefox.java

```diff
@@ -18,7 +20,8 @@ public class TestOpenPageFirefox {
     @Test
     public void openPage() {
         getPropertyManagerInstance();
-        WebDriver driver = WebDriverFactory.getWebDriver(BrowserName.FIREFOX);
+        WebDriver driver = getWebDriverInstance(BrowserName.CHROME);
```

Comment:
remove that line :)

Reply:
@Fishgame1 Basic tests rework to be discussed tomorrow - as per my comment above